File: app/src/main/java/com/xxxifan/smsmaid/AppDatabase.java
package com.xxxifan.smsmaid;

import com.raizlabs.android.dbflow.annotation.Database;

/**
 * Created by BobPeng on 2017/2/13.
 */
@Database(name = AppDatabase.NAME, version = AppDatabase.VERSION)
public class AppDatabase {
    public static final String NAME = "AppDatabase";
    public static final int VERSION = 1;
}

File: app/src/main/java/com/xxxifan/smsmaid/AppDatabase.kt
package com.xxxifan.smsmaid

import com.raizlabs.android.dbflow.annotation.Database

/**
 * Created by BobPeng on 2017/2/10.
 */
@Database(name = AppDatabase.NAME, version = AppDatabase.VERSION)
class AppDatabase {
    companion object {
        const val NAME = "AppDatabase"
        const val VERSION = 1
    }
}
